config/initializers/devise.rb

```diff
@@ -6,7 +7,8 @@
   # ==> Mailer Configuration
   # Configure the e-mail address which will be shown in Devise::Mailer,
   # note that it will be overwritten if you use your own mailer class with default "from" parameter.
-  config.mailer_sender = "GOV.UK Signon <noreply-signon@digital.cabinet-office.gov.uk>"
+  class Dummy; include MailerHelper; end # Disposable class so we can call `email_from`
+  config.mailer_sender = Dummy.new.email_from
```

Comment:
@jamiecobbett: we could make this: `Class.new.extend(MailerHelper).email_from` and avoid the dummy class, not a huge improvement, so feel free to decide whether you want to change it.